src/wallet/wallet.cpp

```diff
@@ -908,6 +911,100 @@ CAmount CWallet::GetChange(const CTransaction& tx) const
     return nChange;
 }
 
+bool CWallet::IsDisabled()
```

Comment:
I don't see a declaration for this, why is it usable in e.g. init.cpp? Just wondering... also could it prepend a ``const``?